includes/payment-methods/class-wc-stripe-upe-payment-method-eps.php

```diff
@@ -19,8 +19,9 @@ class WC_Stripe_UPE_Payment_Method_Eps extends WC_Stripe_UPE_Payment_Method {
 	 */
 	public function __construct( $token_service ) {
 		parent::__construct( $token_service );
-		$this->stripe_id   = self::STRIPE_ID;
-		$this->title       = 'Pay with EPS';
-		$this->is_reusable = true; // TODO: is it?
+		$this->stripe_id            = self::STRIPE_ID;
+		$this->title                = 'Pay with EPS';
+		$this->is_reusable          = true; // TODO: is it?
```

Comment:
```suggestion
		$this->is_reusable          = false;
```
It's not really relevant to this PR, but this should be false (check value for "recurring payments" [here](https://stripe.com/payments/payment-methods-guide#eps)).